mcs/class/corlib/System/TimeZoneInfo.cs

```diff
@@ -925,7 +927,33 @@ public bool IsAmbiguousTime (DateTime dateTime)
 			AdjustmentRule rule = GetApplicableRule (dateTime);
 			if (rule != null) {
 				DateTime tpoint = TransitionPoint (rule.DaylightTransitionEnd, dateTime.Year);
-				if (dateTime > tpoint - rule.DaylightDelta && dateTime <= tpoint)
+				if (dateTime >= tpoint - rule.DaylightDelta && dateTime < tpoint)
+					return true;
+			}
+				
+			return false;
+		}
+        
+		private bool IsAmbiguousLocalDstFromUtc (DateTime dateTime) 
+		{
+			// This method determines if a dateTime in UTC falls into the Dst side
+			// of the ambiguous local time (the local time that occurs twice).
+            
+            if (dateTime.Kind == DateTimeKind.Local)
+                return false;
+
+			if (this == TimeZoneInfo.Utc)
+				return false;
+
+			AdjustmentRule rule = GetApplicableRule (dateTime);
+			if (rule != null) {
+				DateTime tpoint = TransitionPoint (rule.DaylightTransitionEnd, dateTime.Year);
+                // tpoint is the local time in daylight savings time when daylight savings time will end, convert it to UTC
+                DateTime tpointUtc;
+                if (!TryAddTicks(tpoint, -(BaseUtcOffset.Ticks + rule.DaylightDelta.Ticks), out tpointUtc, DateTimeKind.Utc))
```

Comment:
nit: indenting changes within scope. Copy pasted code had a different level of indenting or used tabs maybe?

Reply:
Fixed - tabs vs spaces.